compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -15,1261 +15,13 @@
  */
 
 #include "loader/KernelBuilder.h"
-
```

Comment:
@binarman, thank you =)
For reviewer: related to https://github.com/Samsung/ONE/pull/7614/files#r704665251